pkg/repometrics/stats.go

```diff
@@ -76,12 +197,12 @@ type ReleaseMetrics struct {
 }
 
 const (
-	daysInWeek       = 7
-	daysInMonth      = 30.44
-	daysInYear       = 365.25
-	hoursInDay       = 24
-	roundOneDecimal  = 10
-	roundTwoDecimals = 100
+	daysInWeek  = 7
+	daysInMonth = 30.44
+	daysInYear  = 365.25
+	hoursPerDay = 24
+	hundred     = 100
+	ten         = 10
```

Comment:
I think roundOneDecimal and roundTwoDecimals better conveys what these constants are used for when they are used later on. It is not obvious for everyone from using `ten` in `math.Round(releasesPerDay*ten) / ten` that the idea is to round by one decimal, but I think `math.Round(releasesPerDay*roundOneDecimal) / roundOneDecimal` communicates this well.